### PR TITLE
Add option to control bind port

### PR DESCRIPTION
**Pull request by bararchy**
Were using this internally, thought you might want to merge

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -4,8 +4,11 @@ if [ "$(ls -A /home/firefox/.fonts/)" ]; then
...
+echo "user_pref(\"marionette.port\", $MARIONETTE_PORT);"
+echo "user_pref(\"marionette.port\", $MARIONETTE_PORT);" >> /home/firefox/profile/prefs.js
+
 ip=$(hostname --ip-address)
 
-socat tcp-listen:2828,bind="$ip",fork tcp:127.0.0.1:2828 &
+socat tcp-listen:$MARIONETTE_PORT,bind="$ip",fork tcp:127.0.0.1:$MARIONETTE_PORT &
 
-/usr/bin/firefox -headless -marionette -safe-mode -profile /home/firefox/profile/
+/usr/bin/firefox -headless -marionette -profile /home/firefox/profile/
```
Comment (deepsweet) [inline entrypoint.sh]: why there should be no `-safe-mode` here anymore?

File: profile/prefs.js [hunk 2]
```diff
@@ -74,3 +74,6 @@ user_pref("experiments.manifest.uri", "");
 user_pref("experiments.supported", false);
 user_pref("experiments.activeExperiment", false);
 user_pref("network.allow-experiments", false);
+
+// Marionette settings
+user_pref("marionette.log.level", "debug");
```
Comment (deepsweet) [inline profile/prefs.js]: how does this one looks like? does it spam a lot to the terminal?

File: entrypoint.sh (hunk 1 above)
Comment (bararchy) [reply entrypoint.sh]: Re-Added :)I haven't seen it changes anything too much

File: profile/prefs.js (hunk 2 above)
Comment (bararchy) [reply profile/prefs.js]: Sadly, even with this there are almost no logs :\ makes it really hard to debug Marionette which is already a document-less project

File: entrypoint.sh [hunk 3]
```diff
@@ -4,8 +4,11 @@ if [ "$(ls -A /home/firefox/.fonts/)" ]; then
   fc-cache -f -v
 fi
 
+echo "user_pref(\"marionette.port\", $MARIONETTE_PORT);"
+echo "user_pref(\"marionette.port\", $MARIONETTE_PORT);" >> /home/firefox/profile/prefs.js
+
 ip=$(hostname --ip-address)
 
-socat tcp-listen:2828,bind="$ip",fork tcp:127.0.0.1:2828 &
+socat tcp-listen:$MARIONETTE_PORT,bind="$ip",fork tcp:127.0.0.1:$MARIONETTE_PORT &
```
Comment (deepsweet) [inline entrypoint.sh]: I saw you had changes for default port first but then removed it. I'd like to keep it `2828` by default and have a way to overwrite it only if it's needed. thanks!

**Pull request closed without merging**